domains/Priyanshul.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My Portfolio Website",
+    "repo": "https://github.com/Priyanshul1307/Priyanshul1307.github.io",
+    "owner": {
+      "username": "Priyanshul1307",
+      "email": "piyanshul1307@gmail.com"
+    },
+    "record": {
+      "CNAME": "Priyanshul1307.github.io"

```

Comment:
Site doesn't seem to be accessible, please give a preview of your website

Reply:
sorry for the wrong address for the website its Priyanshul130.github.io
and i have changed the file name to priyanshul.json

Reply:
https://github.com/user-attachments/assets/9cceb502-9a5f-4490-ad3f-34707133275e